open_esplibs/libnet80211/wl_cnx.c

```diff
@@ -0,0 +1,37 @@
+/* Recreated Espressif libnet80211 wl_cnx.o contents.
+
+   Copyright (C) 2015 Espressif Systems. Derived from MIT Licensed SDK libraries.
+   BSD Licensed as described in the file LICENSE
+*/
+#include "open_esplibs.h"
+#if OPEN_LIBNET80211_WL_CNX
+// The contents of this file are only built if OPEN_LIBNET80211_WL_CNX is set to true
+
+#include "espressif/esp_misc.h"
+#include "sdk_internal.h"
+#include <string.h>
+
+void *sdk_cnx_node_search(uint8_t mac[6])
+{
+    int count = sdk_g_ic.s._unknown310 + 2;
```

Comment:
The disassembly did load this slot using `l8ui` so changing to signed would not appear to be appropriate and would cause a sign extension for values over 0x7f. The `count < 1` test does not look right but I don't think it is ever triggered, so until we know more perhaps it's best left as-is, perhaps adding a comment?